charts/opentelemetry-ebpf/templates/k8s-collector-deployment.yaml

```diff
@@ -119,7 +119,7 @@ spec:
           - name: "EBPF_NET_CRASH_METRIC_HOST"
             value: {{ .Values.endpoint.address }}
           - name: "EBPF_NET_CRASH_METRIC_PORT"
-            value: {{ .Values.endpoint.port }}
```

Comment:
Let's take this changes to another PR wdyt?

Reply:
Thanks for review,I'd agree let me move this out, one pr should do one thing